src/vendor-no-composer/robinthehood/ModifiedStdModule/Classes/StdModule.php

```diff
@@ -386,11 +386,26 @@ protected function getConfig(string $name, $default = false): string
      */
     protected function getVersion(): string
     {
-        $tempVersion = $this->tempVersion ?? '';
-        if (!$tempVersion) {
-            $this->tempVersion = $this->getConfig('VERSION');
+        $version = $this->tempVersion ?? '';
+
+        if ('' !== $version) {
+            return $version;
         }
-        return $this->tempVersion;
+
+        $versionConstant = \sprintf('%s_VERSION', $this->modulePrefix);
```

Comment:
I think it's overkill because we can do a simple:

`$versionConstant = $this->modulePrefix . '_VERSION';`